mappings/net/minecraft/world/entity/SectionedEntityCache.mapping

```diff
@@ -0,0 +1,45 @@
+CLASS net/minecraft/class_5573 net/minecraft/world/entity/SectionedEntityCache
```

Comment:
This doesn't behave as just a cache, I think that description might be misleading.

Reply:
I suspect kvverti named it `Cache` because of all the fastutil data structures backing it up, kind of implies speed. this is the class that tracks entities in groups of subchunks at runtime, so idk